namespaces/live.cloud-platform.service.justice.gov.uk/visit-someone-in-prison-backend-svc-dev/resources/irsa.tf

```diff
@@ -1,8 +1,34 @@
-module "app-irsa" {
-  source               = "github.com/ministryofjustice/cloud-platform-terraform-irsa?ref=2.0.0"
+# Add the names of the SQS/SNS which the app needs permissions to access.
+# The value of each item should be the namespace where the SQS/SNS was created.
+# This information is used to collect the IAM policies which are used by the IRSA module.
+locals {
+  sqs_queues = {
+    "Digital-Prison-Services-dev-hmpps_prison_visits_event_queue" = "hmpps_prison_visits_event_queue",
```

Comment:
The values here should be the names of namespaces where the SQS was created - assuming that these SQS queues were created in a different namespace to this one.